examples/todo/src/openapi-spec.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright IBM Corp. 2018,2020. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import {OpenAPIObject} from '@loopback/rest';
+import {TodoListApplication} from './application';
+
+/**
+ * Export the OpenAPI spec from the application
+ */
+async function exportOpenApiSpec(): Promise<OpenAPIObject> {
+  const config: ApplicationConfig = {
+    rest: {
+      port: +(process.env.PORT ?? 3000),
+      host: process.env.HOST ?? 'localhost',
+      openApiSpec: {
+        setServersFromRequest: true,
+      },
+    },
+  };
+  const outFile = process.argv[2] ?? '';
+  const app = new TodoListApplication(config);
+  await app.boot();
+  return app.exportOpenApiSpec(outFile);
+}
+
+if (require.main === module) {
+  exportOpenApiSpec().catch(err => {
+    console.error('Fail to export OpenAPI spec from the application.', err);
```

Comment:
```suggestion
    console.error('Cannot export OpenAPI spec from the application.', err);
```

Reply:
I'll stick with `Fail to` as there is a failure in exporting the spec. IMO, `Cannot` is better when we know why it fails.

Reply:
I am fine with using "fail" instead of "cannot. However, shouldn't this be in the past tense?

```suggestion
    console.error('Failed to export OpenAPI spec from the application.', err);
```

cc @emonddr

Reply:
I think most systems use current tense. For example, `Cannot` instead of `Couldn't`.